src/UID2.Client/DecryptionStatus.cs

```diff
@@ -16,7 +16,7 @@ public enum DecryptionStatus
         ///  DSPs are still expected to check their records for user opt out, even when this status is not returned
         /// </summary>
         UserOptedOut,
-        DomainNameCheckFailed,
+        DomainOrAppNameCheckFailed,
```

Comment:
will this break Adplatform? eg https://gitlab.adsrvr.org/thetradedesk/adplatform/-/blob/master/TTD/Domain/Bidding/Bidder/TTD.Domain.Bidding.Bidder/Identity/IdentityParsing.cs#L351

Reply:
there will be changes required there as we upgrade it to use this version of SDK. tho probably we should check with @vishalegbert-ttd @gmsdelmundo if we change this status, do we need to make further changes in other source codes e.g. spark jobs

Reply:
From the [discussion](https://thetradedesk.slack.com/archives/C051ELQ8JAC/p1714533407971469) in the slack, we can keep this change.